fd/diff.go

```diff
@@ -57,23 +60,30 @@ func Derivative(f func(float64) float64, x float64, settings *Settings) float64
 	if settings == nil {
 		settings = DefaultSettings()
 	}
-	step := settings.Method.Step
+	formula := settings.Formula
+	if formula.Stencil == nil {
```

Comment:
The doc comment should specifically state that under the default settings, the first derivative is estimated.